src/components/Todo/Header1.js

```diff
@@ -0,0 +1,27 @@
+
+import React from 'react'
+
+// // import  './Header1.css';
+// import { AiOutlinePlusCircle } from 'react-icons/ai';
+import Task1 from './Task1';
+import { useState } from 'react';
+
+const Header1 = () => {
+  const [input, setInput]= useState("");
+  const [todo, setTodo]= useState([]);
+
+  return (
+    <div className='header'>
```

Comment:
what is the point of this header1 file you can simply declare the array and on top level ie the todolist.js

Reply:
solve this problem